components/app/footer/FooterBlock.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="lg:flex lg:gap-10">
```

Comment:
suggestion(blocking): dont use flex-gap property since it's not widely supported yet (https://caniuse.com/?search=flex%20gap) The use of grid gap is ok.

This is true for all used flex gap properties - will only comment on this one.